hw2/04-background.js

```diff
@@ -1,13 +1,24 @@
-const interval = document.getElementById('number');
+const interval = document.getElementById('number').value;
 const submit = document.getElementById('submit');
-const background = document.getElementById('background');
+// const background = document.getElementById('background');
 // Add your code here
 
-submit.addEventListener('click', (event) => {
-    setInterval(changeColor, interval*1000);
-})
+// https://stackoverflow.com/questions/1484506/random-color-generator
+const randomColor = function generateRandomRGB() {
```

Comment:
Is this program adding alpha value to your color code? Prof mentioned to add it to dim the intensity and harshness of color. You should check if it's happening!

Reply:
yeah i did but I might dim it more if you think it is an issue

Reply:
Okay, I just couldn't understand the logic behind it, thus wanted to clarify!